src/libs/actions/Transaction.ts

```diff
@@ -10,7 +10,6 @@ import ONYXKEYS from '@src/ONYXKEYS';
 import {RecentWaypoint, Transaction} from '@src/types/onyx';
```

Comment:
Just noting that this is `import type` on `main`, which is something we standardized on in https://github.com/Expensify/App/pull/32475 earlier today. In https://github.com/Expensify/App/pull/33896/files I had the same conflicts and accepted the new way `import type`. But that commit isn't on staging, which is why @marcaaron had the same conflicts here.

This is ok though, we don't need to CP the `import type` change. During our next release cycle we'll delete staging and re-create it from main and end up with the right change on staging 🙂